filesystem/path.h

```diff
@@ -240,6 +241,15 @@ class path {
         return DeleteFileW(wstr().c_str()) != 0;
 #endif
     }
+    
+    bool create_directory() {
+#if defined(WIN32)
+        return SHCreateDirectory(nullptr, make_absolute().wstr().c_str()) == ERROR_SUCCESS;
```

Comment:
It would be best to imitate the behavior of the corresponding function that creates directories in boost::filesystem (which will likely be the chosen API for C++17 and so on..)